### DISCUSSION TITLE
I have issues in the build with vite

### DISCUSSION THREAD

**Discussion by SantiagoRiveros** [vercel/community#1800] [Help] [answered]
### Summary

Hi!

I tried a few times to deploy this project but im not sure whats happening

Anyone has find the solution for this?

### Additional information

<html>
<body>

npm ERR! code ERESOLVE
--
13:52:27.885 | npm ERR! ERESOLVE could not resolve
13:52:27.885 | npm ERR!
13:52:27.885 | npm ERR! While resolving: react-tilt@0.1.4
13:52:27.885 | npm ERR! Found: react@18.2.0
13:52:27.885 | npm ERR! node_modules/react
13:52:27.885 | npm ERR!   react@"^18.2.0" from the root project
13:52:27.886 | npm ERR!   peer react@"^16.8.0 \|\| ^17.0.0 \|\| ^18.0.0" from @react-spring/animated@9.6.1
13:52:27.886 | npm ERR!   node_modules/@react-spring/animated
13:52:27.886 | npm ERR!     @react-spring/animated@"~9.6.1" from @react-spring/core@9.6.1
13:52:27.886 | npm ERR!     node_modules/@react-spring/core
13:52:27.886 | npm ERR!       @react-spring/core@"~9.6.1" from @react-spring/three@9.6.1
13:52:27.886 | npm ERR!       node_modules/@react-spring/three
13:52:27.886 | npm ERR!         @react-spring/three@"~9.6.1" from @react-three/drei@9.57.3
13:52:27.886 | npm ERR!         node_modules/@react-three/drei
13:52:27.886 | npm ERR!     @react-spring/animated@"~9.6.1" from @react-spring/three@9.6.1
13:52:27.886 | npm ERR!     node_modules/@react-spring/three
13:52:27.886 | npm ERR!       @react-spring/three@"~9.6.1" from @react-three/drei@9.57.3
13:52:27.886 | npm ERR!       node_modules/@react-three/drei
13:52:27.886 | npm ERR!         @react-three/drei@"^9.56.24" from the root project
13:52:27.887 | npm ERR!   17 more (@react-spring/core, @react-spring/shared, ...)
13:52:27.887 | npm ERR!
13:52:27.887 | npm ERR! Could not resolve dependency:
13:52:27.887 | npm ERR! peer react@"^15.0.0 \|\| ^16.0.0-beta \|\| ^16.0.0" from react-tilt@0.1.4
13:52:27.887 | npm ERR! node_modules/react-tilt
13:52:27.887 | npm ERR!   react-tilt@"^0.1.4" from the root project
13:52:27.887 | npm ERR!
13:52:27.887 | npm ERR! Conflicting peer dependency: react@16.14.0
13:52:27.887 | npm ERR! node_modules/react
13:52:27.887 | npm ERR!   peer react@"^15.0.0 \|\| ^16.0.0-beta \|\| ^16.0.0" from react-tilt@0.1.4
13:52:27.887 | npm ERR!   node_modules/react-tilt
13:52:27.887 | npm ERR!     react-tilt@"^0.1.4" from the root project
13:52:27.887 | npm ERR!
13:52:27.887 | npm ERR! Fix the upstream dependency conflict, or retry
13:52:27.887 | npm ERR! this command with --force, or --legacy-peer-deps
13:52:27.888 | npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
13:52:27.888 | npm ERR!
13:52:27.888 | npm ERR! See /vercel/.npm/eresolve-report.txt for a full report.
13:52:27.888 |  
13:52:27.888 | npm ERR! A complete log of this run can be found in:
13:52:27.888 | npm ERR!     /vercel/.npm/_logs/2023-03-16T16_52_27_360Z-debug-0.log
13:52:27.906 | Error: Command "npm install" exited with 1
13:52:28.123 | BUILD_FAILED: Command "npm install" exited with 1

</body>
</html>

**Comment by amyegan** [comment, marked as answer]
It looks like your project is having trouble installing the right packages because it has a conflicting peer dependency. In that case, you may need to update one or more of your dependencies to get the peer dependencies to agree. Or you can update your install command to use `--force` or `--legacy-peer-deps`
